rand_core/src/block.rs

```diff
@@ -233,7 +234,8 @@ where <R as BlockRngCore>::Results: AsRef<[u32]> + AsMut<[u32]>
 
     #[inline(always)]
     fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), Error> {
-        Ok(self.fill_bytes(dest))
+        self.fill_bytes(dest);
+        Ok(())
```

Comment:
?

Reply:
This change is to appease a [lint intended to catch accidental semicolons](https://rust-lang.github.io/rust-clippy/master/index.html#unit_arg), so arguably a false positive.

Reply:
Should I revert this? I think the new version is slightly more clear, but I don't have a strong preference.